lib/ansible/modules/cloud/amazon/aws_waf_regional_rule.py

```diff
@@ -0,0 +1,319 @@
+#!/usr/bin/python
+# Copyright (c) 2017 Will Thames
+# Copyright (c) 2015 Mike Mochan
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+module: aws_waf_regional_rule
+short_description: create and delete WAF Regional Rules
+description:
+  - Read the AWS documentation for WAF Regional
+    U(https://aws.amazon.com/documentation/waf/)
+version_added: "2.6"
+
+author:
+  - Mike Mochan (@mmochan)
+  - Will Thames (@willthames)
+  - Jaco Engelbrecht (@bje)
+extends_documentation_fragment:
+  - aws
+  - ec2
+options:
+    name:
+        description: Name of the Web Application Firewall rule
+        required: yes
+    metric_name:
+        description:
+        - A friendly name or description for the metrics for the rule
+        - The name can contain only alphanumeric characters (A-Z, a-z, 0-9); the name can't contain whitespace.
+        - You can't change metric_name after you create the rule
+        - Defaults to the same as name with disallowed characters removed
+    state:
+        description: whether the rule should be present or absent
+        choices:
+        - present
+        - absent
+        default: present
+    conditions:
+        description: >
+          list of conditions used in the rule. Each condition should
+          contain I(type): which is one of [C(byte), C(geo), C(ip), C(size), C(sql) or C(xss)]
+          I(negated): whether the condition should be negated, and C(condition),
```

Comment:
```suggestion
          I(negated) whether the condition should be negated, and C(condition),
```

The `: ` is what's breaking yaml parsing